index.html

```diff
@@ -1,181 +1,209 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles.css" />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap"
-      rel="stylesheet"
-    />
-    <title>Chiedozie's Portfolio Website</title>
-  </head>
-  <body>
-    <header>
-      <nav class="fluidsContainer">
-        <div class="navContainer">
-          <div class="navbar">
-            <div class="logo">
-              <a href="#" class="my-logo">James Chiedozie James</a>
-            </div>
-            <div>
-              <i class="fas fa-bars"></i>
-            </div>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+  <link rel="stylesheet" href="styles.css" >
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" >
+  <title>Chiedozie's Portfolio Website</title>
+</head>
+
+<body>
+  <header>
```

Comment:
- [ ] Well done on creating the toolbar section! However, I noticed that the navigation items in your code do not match the design guidelines. Please carefully review the design and update your code to ensure that the navigation items are styled according to the specified design. Pay attention to aspects such as font styles, spacing, and alignment to achieve a visually consistent and cohesive navigation menu. Keep up the great work! 👍🎉


| Your | Professional |
| --- | --- |
|  ![image](https://github.com/chiedozie013/portfolio-microverse/assets/55840999/e12995e4-4e8d-4ad6-b5af-44ef2c654815)  |  ![image](https://github.com/chiedozie013/portfolio-microverse/assets/55840999/666cde16-08e6-4de3-beb9-25ac608650a3)    |